misc/io.github.grangerhub.Tremulous.appdata.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2017 Daniel Rusek <mail@asciiwolf.com> -->
+<component type="desktop-application">
+  <id>io.github.grangerhub.Tremulous</id>
+  <project_license>GPL-2.0</project_license>
+  <metadata_license>CC-BY-SA-3.0</metadata_license>
+  <provides>
+    <id>io.github.grangerhub.Tremulous.desktop</id>
+  </provides>
+  <launchable type="desktop-id">io.github.grangerhub.Tremulous.desktop</launchable>
+  <name>Tremulous</name>
+  <developer_name>Dark Legion Development and GrangerHub</developer_name>
+  <summary>Aliens vs Humans, First Person Shooter game with elements of Real Time Strategy (Grangerhub version)</summary>
+  <description>
+    <p>
+      Tremulous is a free, open source game that blends a team based FPS with
+      elements of an RTS.
+    </p>
+    <p>
+      Players can choose from 2 unique races, aliens and humans.
+      Players on both teams are able to build working structures in-game like an
+      RTS. 
+    </p>
+  </description>
+  <url type="homepage">https://github.com/GrangerHub/tremulous</url>
+  <screenshots>
```

Comment:
One last small thing: We should consider adding more screenshots with aliens. Currently, most of them are with humans. :)

Reply:
Added a bunch! About to commit.

Reply:
Thanks! We should probably also focus more on quality than quantity and keep only the good screenshot + remove screenshots that have different resolution and look inconsistent.

Reply:
> Thanks! We should probably also focus more on quality than quantity and keep only the good screenshots + remove screenshots that have different resolution and look inconsistent.

I already removed the low res screenshots from the appdata file. I have some extra screenshots left in the folder, but I'd like to keep them for posterity. They are not referenced from the appdata file.